go.mod

```diff
@@ -4,10 +4,13 @@ go 1.23.0
 
 toolchain go1.24.1
 
-require gopkg.in/yaml.v3 v3.0.1
+require (
+	github.com/goccy/go-yaml v1.17.1
+	github.com/stretchr/testify v1.10.0
+)
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/stretchr/testify v1.10.0 // indirect
+	gopkg.in/yaml.v3 v3.0.1 // indirect
```

Comment:
bit of a laugh that using testify still means this is still getting imported

Reply:
https://github.com/stretchr/testify/pull/1725

Reply:
And, I'll gladly trade the direct dep for an indirect